### DISCUSSION TITLE
Requesting Sponsership- Counterspell Abu Dhabi

### DISCUSSION THREAD

**Discussion by akulsaju** [community/community#140406] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

We would love to have you as a sponsor for our event. This could be a great way to increase the brand visibility of your company among teens around the world with incredible technical abilities (see some of the open-source tools built by teens at [Hack Club](https://hackclub.com/); Hack Clubbers will be the primary source of attendees at the event). A contribution at any level will help. Since we all do this work voluntarily, your contribution will go directly towards creating a magical event for our attendees.

Your contributions will be tax-deductible, as we qualify as a 501(c)(3) being fiscally sponsored by The Hack Foundation (EIN: 81-2908499).

I have attached the prospectus below.
[Uploading Sponsor Prospectus.pdf…]()

**Comment by ebndev** [comment]
Hi @akulsaju 👋

Thanks for posting in the Community! The `Sponsors` category is a place for our community to discuss and provide feedback on the GitHub Sponsors feature, not for sponsorship opportunities. 

Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations.

We appreciate your understanding!
